### DISCUSSION TITLE
Disable Symphony web toolbar in frontend

### DISCUSSION THREAD

**Discussion by incoxx** [pimcore/pimcore#10037] [General]
**Expected behavior**
Using a Controller and a Twig template, i would like to render a page WITHOUT the symphony toolbar code at the bottom

**Actual behavior**
At the end of the rendered HTML page, i found the symphony toolbar code before </body>

**Sample before the </body> tag:**
<div id="sfwdtb0a2a5" class="sf-toolbar sf-display-none" role="region" aria-label="Symfony Web Debug Toolbar"></div>

**My question:**
How can i disable the Symphony Web Debug Toolbar in PIMCORE X? I took a look in several config files, but i have no idea how to disable the toolbar

**Comment by incoxx** [comment]
Sample at the end of the HTML page before the `</body>` tag
 `<div id="sfwdtb0a2a5" class="sf-toolbar sf-display-none" role="region" aria-label="Symfony Web Debug Toolbar"></div>.........`

**Reply by fashxp** [reply]
change environment to production?

**Reply by incoxx** [reply]
Hi Christian, yes, i have no idea to change it in pimcore X

**Reply by fashxp** [reply]
change it in `.env` file ... see also https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html ... Pimcore X now follows Symfony standard

**Reply by jdreesen** [reply]
https://symfony.com/doc/current/reference/configuration/web_profiler.html#toolbar

**Reply by incoxx** [reply]
Dear Christian and Jacob, thanks a lot.

this was the missing part

![image](https://user-images.githubusercontent.com/21988262/129930165-6678aa7a-cd7e-40a5-9756-750163c05626.png)

The file is invisible in putty ... that´s the problem :)

Thank you!
Markus
